.lgmt.yaml

```diff
@@ -0,0 +1,20 @@
+path_classifiers:
+  test:
+    - test
+    - scripts
+
+extraction:
+  cpp:
+    prepare:
+      packages:
+        - python-pip
+    after_prepare:
+      - export PATH=~/.local/bin:$PATH
+      - pip3 install cmake --user
+      - cmake --version
+    configure:
+      command:
+        - cmake -S . -B build -DBUILD_QECC_TESTS=ON -DBINDINGS=ON
+    index:
+      build_command:
+        - cmake --build build --parallel 4
```

Comment:
this file can be removed as LGTM is deprecated and will be shut down at the end of the year